SEP-0002.md

```diff
@@ -3,203 +3,288 @@
-non-profit organizations (e.g.
-[501c](http://en.wikipedia.org/wiki/501(c)_organization). This
-organization structure is tried and tested and has the added advantage
-that it should provide a smooth transition to incorporating a SunPy
-foundation if it becomes necessary in the future (e.g. for funding
-purposes). The primary role of the organization is to *facilitate and
-promote the use and development of community-led, free and
-open-source solar data-analysis software based on the scientific
-Python environment*. This includes but is not limited to the following
-tasks
-
-* developing the SunPy software library
-* manage and protect the SunPy
-brand and identity
-* promote SunPy to the community
-* managing SunPy assets, i.e. github account, domain name etc.
-
-The organization consists of an executive director, an advisory board, and the
-developer community. The goal of the proposed structure is to provide and
-promote community input into the project.
+| discussion    | <https://github.com/sunpy/sunpy-SEP/pull/62>            |
+| status        | in review                                               |
+
+## Rationale
+
+This SEP describes the second major iteration of the governance for the SunPy Project.
+The objectives of the rewrite were to adapt the governance for the changes the project has seen since the first iteration of this document was written in 2014.
+The rewrite aimed to detail a governance structure, which strives to be lightweight, while being designed to enable the project to grow, and facilitating more input from the wider SunPy community.
+
+## The SunPy Project
+
+The primary mission of the SunPy Project is **To develop and facilitate a high-quality, easy-to-use, community-led, free and open-source solar data analysis ecosystem based on the scientific Python environment**.
+This includes but is not limited to the following tasks:
+
+* developing the sunpy core library and other affiliated packages
+* supporting a community of affiliated packages
+* managing and protecting the SunPy brand and identity
+* promoting SunPy to the community
+* managing SunPy assets, e.g. GitHub organization, domain name etc.
+
+## Governance Structure
+
+The SunPy Project's governance involves the following five components:
+
+* The Community
+* The Project Custodians
+* The Steering Committee
+* The Advisory Board
+* The Ombudsperson
+
+The SunPy project exists for the benefit of the whole Community: users, developers, and members of the wider solar physics and scientific software community.
+The Community is the ultimate authority in how the project is managed.
+This authority is exercised through the processes outlined in this document, primarily through the Steering Committee and the Project Custodians.
+Whenever possible, decisions shall be approved by consensus, considering input from the Community, rather than taken unilaterally by Project Custodians or the Steering Committee.
```

Comment:
first cut:
"When possible and necessary, input will be gathered from the Community through multiple input channels, including online polling, direct communication, and in person interactions at online gatherings. These input channels will be actively used to reach decisions by consensus, rather than taken unilaterally by Project Custodians or the Steering Committee."

Reply:
```suggestion
When feasible and necessary, input will be gathered from the Community through multiple channels, for example forums, online polling, direct communication, and in-person interactions at domain-related meetings.
Wherever possible, these input channels will be actively used to reach decisions by consensus, rather than taken unilaterally by Project Custodians or the Steering Committee.
```